x-pack/legacy/plugins/ml/server/routes/data_frame_analytics.ts

```diff
@@ -0,0 +1,288 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { schema } from '@kbn/config-schema';
+import { analyticsAuditMessagesProvider } from '../models/data_frame_analytics/analytics_audit_messages';
+import { licensePreRoutingFactory } from '../new_platform/licence_check_pre_routing_factory';
+import { RouteInitialization } from '../new_platform/plugin';
+import {
+  dataAnalyticsJobConfigSchema,
+  dataAnalyticsEvaluateSchema,
+  dataAnalyticsExplainSchema,
+} from '../new_platform/data_analytics_schema';
+
+export function dataFrameAnalyticsRoutes({ xpackMainPlugin, router }: RouteInitialization) {
+  router.get(
+    {
+      path: '/api/ml/data_frame/analytics',
+      validate: {
+        params: schema.object({ analyticsId: schema.maybe(schema.string()) }),
+      },
+    },
+    licensePreRoutingFactory(xpackMainPlugin, async (context, request, response) => {
+      try {
+        const results = await context.ml!.mlClient.callAsCurrentUser('ml.getDataFrameAnalytics');
+        return response.ok({
+          body: { ...results },
+        });
+      } catch (e) {
+        return response.forbidden({ body: e.message });
```

Comment:
is this going to return a 403 for every error that gets thrown.
Do we not want to do something similar to how the old routes work where the errors thrown from the endpoints are wrapped in a boom error? or something similar if we don't want to use boom anymore.

Reply:
Yep - find and replace gone wrong which I thought I removed but slipped through. Thanks. Fixed in 8c93d6b4a20f1de2821b6ab6f8835703bdb09abf